.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.parcel-cache
```

Comment:
yeah, we are including parcel now, this is only part of the build, and dev-dependencies (so consuming project won't have to rely on having parcel).